integrations/resend/src/index.ts

```diff
@@ -1,17 +1,44 @@
 import type { IntegrationClient, TriggerIntegration } from "@trigger.dev/sdk";
 import { Resend as ResendClient } from "resend";
 
+
 import type { AuthenticatedTask } from "@trigger.dev/sdk";
 
 type SendEmailData = Parameters<InstanceType<typeof ResendClient>["sendEmail"]>[0];
 
 type SendEmailResponse = { id: string };
 
+function isRequestError(error: unknown): boolean {
+  return typeof error === "object" && error !== null && "status" in error;
+}
+
+function onError(error: any) {
```

Comment:
The types for this are only working because you are typing the error as any here, it should be changed to `unknown` and the type guard above (`isRequestError`) should correctly guard to the error type in the Resend SDK.